account_invoice_pricelist/model/account_invoice.py

```diff
@@ -1,58 +1,29 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Account - Pricelist on Invoices for Odoo
-#    Copyright (C) 2015-Today GRAP (http://www.grap.coop)
-#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from openerp import models, fields, api
 
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    # Column Section
     pricelist_id = fields.Many2one(
         comodel_name='product.pricelist', string='Pricelist',
         help="The pricelist of the partner, when the invoice is created"
         " or the partner has changed. This is a technical field used"
-        " for reporting.")
+        " for reporting.",
```

Comment:
Remove last part of the help, as this is not true anymore (it's not a technical field).